tasks/main.yml

```diff
@@ -2,6 +2,9 @@
 - name: Gather variables for each operating system
```

Comment:
This is no longer needed since `tasks/selinux.yml` is already discovering and installing necessary dependencies.